CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Our versioning strategy is as follows:
 - Major: may include breaking changes in core packages (e.g. major architectural changes, major features)
 
 ## Unreleased
+* `[templates/nextjs-sxa]` Add scaffold components action `jss scaffold {nameOfComponent}` ([#1420](https://github.com/Sitecore/jss/pull/1420))
```

Comment:
@matkovskyi  It should be added under specific section: Bug Fix, Improvements - you can see example in a same file below.
In your case it's improvement